test/chi/custom-elements/date-picker.pug

```diff
@@ -26,3 +26,8 @@ scripts:
   span.a-h2 Input Popover Combined w/ icon
   .-position--relative(style="height: 18rem; width:14rem;")
     chi-date-picker(value=date2, min=date2, active)
+
+.test-disabled(data-cy="test-disabled")
+  span.a-h2 Disabled date picker
+  .-position--relative(style="width:14rem;")
+    chi-date-picker(disabled="true")
```

Comment:
Question: is true value explicitly needed here?

Reply:
I tried  `chi-date-picker(disabled)` and it works fine.
All done, thanks for the suggestions!